Tools/SwiftBrowser/Source/ContentView.swift

```diff
@@ -24,6 +24,7 @@
 import SwiftUI
 import UniformTypeIdentifiers
 @_spi(Private) import WebKit
+@_spi(Private) import _WebKit_SwiftUI
```

Comment:
What does this do? Seems like we should already get the cross-import in this file, because of the SwiftUI and WebKit imports. And the cross-import module does not itself have any `@_spi(Private)` declared.

Reply:
when using SPI, it's necessary to use the cross-import name explicitly. And _WebKit_SwiftUI does have several `@_spi(Private)` symbols currently (such as webViewAllowsBackForwardNavigationGestures)

Reply:
I see, it was hiding outside the diff. Thanks!